src/RestaurantsContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { useSelector } from 'react-redux';
+
+import Restaurants from './Restaurants';
+
+export default function RestaurantsContainer() {
+  const { selectedRegionId, selectedCategoryId, restaurants } = useSelector((state) => ({
+    selectedRegionId: state.selectedRegionId,
+    selectedCategoryId: state.selectedCategoryId,
+    restaurants: state.restaurants,
+  }));
+
+  if (!selectedRegionId || !selectedCategoryId) {
+    return <p>지역과 분류를 선택해주세요!</p>;
+  }
+
+  return (
+    <div>
+      <h4>레스토랑 목록</h4>
+      <Restaurants restaurants={restaurants} />
+    </div>
+  );
```

Comment:
category와 region이 선택되지 않은 경우에 대한 처리를 해주셨네요 👍 

다만 부정문을 사용하는것보다 가능하면 긍정문으로(not을 사용하지 않는) 처리해보면 좋을 것 같아요~

```suggestion
  return (
    {  selectedRegionId && selectedCategoryId ? (
      <>
        <h4>레스토랑 목록</h4>
        <Restaurants restaurants={restaurants} />
      </>
    : (
      <p>지역과 분류를 선택해주세요!</p>
    )
  );
```

Reply:
부정문을 사용하는것보다 가능하면 긍정문으로(not을 사용하지 않는) 처리하는게 가독성면에서도 좋은 것 같아요!!